desc/compute/_profiles.py

```diff
@@ -52,7 +52,7 @@ def _psi_r(params, transforms, profiles, data, **kwargs):
     params=["Psi"],
     transforms={},
     profiles=[],
-    coordinates="",
+    coordinates="r",
```

Comment:
This is a constant, so it could? be either an empty string or all coordinates.

Reply:
Same as above